deploy-gatsby.md

```diff
@@ -0,0 +1,28 @@
+# Deploying Gatsby
+
+## Best Practice
+
+
+Though you can deploy from the same location multiple times it is recommended that you clear your public directory before each build
+e.g. using surge
+
+```bash
+rm -rf public/ && gatsby build && surge public/
+```
+
+because this is going to be executed on every deploy it is suggested that you use an `package.json` script to simplify this process
+
+
+## Providers
```

Comment:
Perhaps just link to a number of static hosts? There's no reason to prioritize Surge + it'll be perhaps hard for us to keep installation instructions up-to-date. Better I'd think to just send people off the hosting site?